### ISSUE TITLE
Spacing between the quiz box and other elements

### ISSUE THREAD

**Issue by ishikagoyal02** [srbmaury/quiz1#49]
There is no space between the user name and the quiz box. Similarly, with the timer, the UI of the quiz is not good. Assign me this issue 

![image](https://github.com/srbmaury/quiz1/assets/99865224/4210d805-d17e-4692-be34-2ed52038456b)
![image](https://github.com/srbmaury/quiz1/assets/99865224/81e923f7-0e11-40ac-8a3e-9ed617ff496f)

**Comment by love-sonkar** [comment]
hey @ishikagoyal02 you can fork and start the work because sometimes the maintainer is busy so he didn't assign or he forgot to assign so it's better to fork and start working in issue

**Comment by ishikagoyal02** [comment]
Ok thanks

**Comment by love-sonkar** [comment]
yup, otherwise someone else pick this issue so it's better make an issue and after 2 3 hour's start the work
